app/components-react/root/LiveDock.tsx

```diff
@@ -364,115 +334,93 @@ function LiveDock(p: { onLeft: boolean }) {
+            <i
+              className={cx({
+                ['icon-view']: !hideViewerCount,
+                ['icon-hide']: hideViewerCount,
+              })}
+              onClick={() => ctrl.toggleViewerCount()}
+            />
+            <span className={styles.liveDockViewerCountCount}>{viewerCount}</span>
+            {Number(viewerCount) >= 0 && <span>{$t('viewers')}</span>}
+          </div>
+        </div>
+
+        <div className={styles.liveDockInfo}>
+          <div className={styles.liveDockPlatformTools}>
+            {ctrl.canEditChannelInfo && (
+              <Tooltip
+                title={$t('Edit your stream title and description')}
+                placement="right"
+                autoAdjustOverflow={false}
+              >
+                <i onClick={() => ctrl.showEditStreamInfo()} className="icon-edit" />
+              </Tooltip>
+            )}
+            {isPlatform(['youtube', 'facebook', 'trovo', 'tiktok', 'kick']) && isStreaming && (
+              <Tooltip
+                title={$t('View your live stream in a web browser')}
+                placement="right"
+                autoAdjustOverflow={false}
+              >
+                <i onClick={() => ctrl.openPlatformStream()} className="icon-studio" />
+              </Tooltip>
+            )}
+            {isPlatform(['youtube', 'facebook', 'tiktok']) && isStreaming && (
+              <Tooltip
+                title={$t('Go to Live Dashboard')}
+                placement="right"
+                autoAdjustOverflow={false}
+              >
+                <i onClick={() => ctrl.openPlatformDash()} className="icon-settings" />
+              </Tooltip>
             )}
           </div>
-        )}
-      </Animation>
+          <div className="flex">
+            {(isPlatform(['twitch', 'trovo', 'facebook', 'kick']) ||
+              (isPlatform(['youtube', 'twitter']) && isStreaming) ||
+              (isPlatform(['tiktok']) && isRestreaming)) && (
+              <a onClick={() => ctrl.refreshChat()}>{$t('Refresh Chat')}</a>
+            )}
+          </div>
+        </div>
+        {!hideStyleBlockers &&
+          (isPlatform(['twitch', 'trovo']) ||
+            (isStreaming && isPlatform(['youtube', 'facebook', 'twitter', 'tiktok', 'kick']))) && (
+            <div className={styles.liveDockChat}>
+              {hasChatTabs && <ChatTabs visibleChat={visibleChat} setChat={setChat} />}
+              {!applicationLoading && !collapsed && chat}
+              {!['default', 'restream'].includes(visibleChat) && (
+                <PlatformAppPageView
+                  className={styles.liveDockPlatformAppWebview}
+                  appId={visibleChat}
+                  pageSlot={pageSlot}
+                  key={visibleChat}
+                />
+              )}
+            </div>
+          )}
+      </div>
+      {(!ctrl.platform ||
+        (isPlatform(['youtube', 'facebook', 'twitter', 'tiktok', 'kick']) && !isStreaming)) && (
```

Comment:
not really a comment, but a note to self, lots of `isPlatform` evaluations over here, most of the time with the same arguments, that should probably be extracted in the future. It will make it easier to support newer platforms as well instead of having to track all these instances

Reply:
i think that's slightly out of scope for this PR but yes agreed, i have a thought to handle this similar how we use `hasCapability` for platform features

Reply:
made a follow up PR here for when this is merged
https://github.com/streamlabs/desktop/pull/5401